src/codegen/sys/lib_.rs

```diff
@@ -489,7 +497,7 @@ fn generate_from_fields(
         // It would be nice to represent those using extern types
         // from RFC 1861, once they are available in stable Rust.
         // https://github.com/rust-lang/rust/issues/43467
-        writeln!(w, "pub struct {name}(c_void);", name = &fields.name)?;
```

Comment:
Do we still use `c_void` for something? If not we can remove it from the imports

Reply:
I don't think we do. Let's try removing it. :)

Reply:
```
error[E0412]: cannot find type `c_void` in this scope
  --> gdk/sys/src/lib.rs:35:22
   |
35 | pub type GdkXEvent = c_void;
   |                      ^^^^^^ not found in this scope
```

So not yet!